### PR TITLE
Form translations

### PR DESCRIPTION
**Pull request by davidrapson**
Add Welsh language translations for Awards for All form

### CODE REVIEW

**Comment by NikhilNanjappa** [comment]
Assuming all the `form.js` translations are in the doc provided, there were few of the translations missing which I've listed down:

`controllers/apply/award-for-all/form.js`:
Line 667
Line 776
Line 832
Line 1130
Lin 1171
Line 1211
Line 1226

`controllers/apply/award-for-all/confirmation.js`:
Line 86

File: controllers/apply/awards-for-all/form.js [hunk 1]
```diff
@@ -799,7 +826,12 @@ module.exports = function({
                             We can't transfer money into certain types of bank
                             accounts like Tide, Cashplus and Paypal.
                         </p>`,
-                        cy: ''
+                        cy: `<p>
+                            Rydym angen eich manylion banc i dalu’r arian i’ch cyfrif – os yw eich cais yn llwyddiannus. 
+                        </p>
+                        <p><strong>We can't pay into all bank accounts</strong>:
```
Comment (NikhilNanjappa) [inline controllers/apply/awards-for-all/form.js]: This line was left as English in the Welsh translation ie. "We can't pay into all bank accounts" was not translated.

File: controllers/apply/awards-for-all/form.js [hunk 2]
```diff
@@ -87,15 +87,20 @@ module.exports = function({
                     <p><a href="/funding/programmes/awards-for-all-northern-ireland">You'll need to download an application form to apply</a>.</p>
                     <p>Only applicants in Scotland can apply through our new online form at the moment. We're working on making this available for the rest of the UK.</p>
                 `,
-                cy: ``
+                cy: `
+                    <p><a href="https://apply.tnlcommunityfund.org.uk">Byddwch angen defnyddio’r ffurflen hon yn lle</a>.</p>
+                    <p><strong>Ymgeisio am brosiect yng Ngogledd Iwerddon?</strong></p>
+                    <p><a href="/welsh/funding/programmes/awards-for-all-northern-ireland">Byddwch angen lawrlwytho ffurflen gais i ymgeisio</a>.</p>
```
Comment (NikhilNanjappa) [inline controllers/apply/awards-for-all/form.js]: Highlighting the hyperlink change to make sure it exists.

File: controllers/apply/awards-for-all/form.js [hunk 3]
```diff
@@ -867,15 +899,64 @@ module.exports = function({
...
+                ond mae lluniau o’ch cyfriflenni banc yn hollol iawn.
+            </strong></p>
+            
+            <div class="o-media u-padded u-tone-background-tint u-margin-bottom">
+                <a href="../help/bank-statement" target="_blank">
+                    <img src="/assets/images/apply/afa-bank-statement-example-small.png"
+                         alt="Enghraifft o gyfriflen banc rydym ei angen gennych"
+                         class="o-media__figure-gutter"
+                         width="300" />
+                    <span class="u-visually-hidden">Yn agor mewn ffenest newydd</span>
```
Comment (NikhilNanjappa) [inline controllers/apply/awards-for-all/form.js]: Alternate text & visually hidden text of the link translated.

File: controllers/apply/awards-for-all/form.js [hunk 4]
```diff
@@ -908,20 +994,34 @@ module.exports = function({
...
+                a fydd yn ei ddefnyddio i atal twyll a gwyngalchu arian, 
+                ac i wireddu eich hunaniaeth. Os oes twyll yn cael ei ganfod, 
+                gellid eich gwrthod rhag rhai gwasanaethau, arian neu gyflogaeth.
+            </p>
+            <p>
+                Gall gwybodaeth pellach ar sut bydd eich gwybodaeth yn cael ei ddefnyddio gennym a’r 
+                asiantaethau atal twyll hyn, eich hawliau gwarchod data a sut i gysylltu â ni, ei ganfod yn ein 
+                Hysbysiad Diogelu Data a Phreifatrwydd llawn, sydd wedi’i gyhoeddi ar ein gwefan 
+                <a href="/welsh/about/customer-service/data-protection">https://www.tnlcommunityfund.org.uk/welsh/about/customer-service/data-protection</a>. 
+                Cysylltwch â ni i ofyn am gopi caled drwy ffonio ein llinell gynghori ar 0345 4 10 20 30,
```
Comment (NikhilNanjappa) [inline controllers/apply/awards-for-all/form.js]: Highlighting the hyperlink change to make sure it exists.

File: controllers/apply/awards-for-all/form.js (hunk 2 above)
Comment (mattandrews) [reply controllers/apply/awards-for-all/form.js]: That one doesn't (we don't need to translate country-specific content that isn't Wales-related – so this link should just be `/funding/programmes/awards-for-all-northern-ireland`

File: controllers/apply/awards-for-all/form.js (hunk 4 above)
Comment (mattandrews) [reply controllers/apply/awards-for-all/form.js]: Yep this one is right!

File: controllers/apply/awards-for-all/form.js (hunk 2 above)
Comment (NikhilNanjappa) [reply controllers/apply/awards-for-all/form.js]: Got it! Have changed this, will push it with the next commit

File: controllers/apply/awards-for-all/fields.js [hunk 5]
```diff
@@ -2512,10 +2297,10 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             type: 'checkbox',
             label: localise({
                 en: `If information about this application is requested under the Freedom of Information Act, we will release it in line with our <a href="/about/customer-service/freedom-of-information">Freedom of Information policy.</a>`,
-                cy: ''
+                cy: 'Os gofynnir am wybodaeth o’r cais hwn o dan y Ddeddf Rhyddid Gwybodaeth, byddwn yn ei ryddhau yn unol â’n <a href="/welsh/about/customer-service/freedom-of-information">Polisi Rhyddid Gwybodaeth.</a>'
```
Comment (NikhilNanjappa) [inline controllers/apply/awards-for-all/fields.js]: Hyperlinks changed to its welsh counterpart.

File: controllers/apply/awards-for-all/fields.js [hunk 6]
```diff
@@ -2489,10 +2274,10 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             type: 'checkbox',
             label: localise({
                 en: `You understand that we will use any personal information you have provided for the purposes described under the <a href="/about/customer-service/data-protection">Data Protection Statement</a>.`,
-                cy: ''
+                cy: 'Rydych yn deall y byddwn yn defnyddio unrhyw wybodaeth bersonol rydych wedi ei ddarparu ar gyfer dibenion wedi’i ddisgrifio dan y <a href="/welsh/about/customer-service/data-protection">Datganiad Diogelu Data</a>.'
```
Comment (NikhilNanjappa) [inline controllers/apply/awards-for-all/fields.js]: Hyperlink changed to its welsh counterpart.

**Comment by davidrapson** [comment]
Worth noting that I started to pull some of the field definitions out into their own files as `fields.js` was getting unwieldy. Limited it to the thornier fields (org type, roles, ideas questions) but we should come back and extract all the fields out as a clean up task after this.

**Comment by davidrapson** [comment]
Made a couple of final tweaks to enable the language switcher in the test environment and fixup a couple of missing bits but otherwise I think we are good to go.

File: controllers/apply/awards-for-all/fields.js [hunk 7]
```diff
@@ -1401,133 +1084,155 @@ module.exports = function fieldsFor({ locale, data = {} }) {
...
+                                en: 'Bangladeshi',
+                                cy: 'Bangladeshi'
+                            })
                         },
                         {
                             value: 'chinese',
-                            label: localise({ en: 'Chinese', cy: '' })
+                            label: localise({
+                                en: 'Chinese',
+                                cy: 'Tsieniaidd '
```
Comment (mattandrews) [inline controllers/apply/awards-for-all/fields.js]: Trailing space here

File: controllers/apply/awards-for-all/fields.js [hunk 8]
```diff
@@ -1401,133 +1084,155 @@ module.exports = function fieldsFor({ locale, data = {} }) {
...
-                        cy: ''
+                        cy: 'Du / Affricanaidd / Caribiaidd / Du Brydeinig'
                     }),
                     options: [
                         {
                             value: 'caribbean',
-                            label: localise({ en: 'Caribbean', cy: '' })
+                            label: localise({
+                                en: 'Caribbean',
+                                cy: 'Caribiaidd '
```
Comment (mattandrews) [inline controllers/apply/awards-for-all/fields.js]: Trailing space (and below too)

File: controllers/apply/awards-for-all/form.js (hunk 1 above)
Comment (mattandrews) [reply controllers/apply/awards-for-all/form.js]: TODO – we should fix this before merging as we'll definitely forget :D

Comment (davidrapson) [reply controllers/apply/awards-for-all/form.js]: Have just requested this. Worked out why this was missed, the text was emboldened and we asked for bold text to be ignored :)

**Pull request merged**